### PR TITLE
docker_swarm_service: rename return variable to swarm_service

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
As discussed in #51939, renaming the return variable to `swarm_service`.

##### ISSUE TYPE
- Bugfix Pull Request
- Docs Pull Request

##### COMPONENT NAME
docker_swarm_service

### CODE REVIEW

**Comment by felixfontein** [comment]
@hannseman @jwitko please say something if you don't like this change :) (the new name was suggested by @dariko, see [here and below](https://github.com/ansible/ansible/pull/51939#issuecomment-469015852)). Or `shipit` it if you like it :)

**Review by hannseman — APPROVED** [review]
LGTM

**Comment by felixfontein** [comment]
@abadger I assume you're still the release manager for Ansible 2.7.x. Can you take a look at this PR and say if it is ok from your point of view? Otherwise it doesn't make sense to merge it in this form (and backport it to stable-2.7).

**Review by jwitko — APPROVED** [review]
LGTM

**Comment by felixfontein** [comment]
After talking to @abadger, I've added a remark that the old name will stay for Ansible 2.7.x. I will add that in the backport PR once this is merged.

**Comment by felixfontein** [comment]
ready_for_review

**Comment by WojciechowskiPiotr** [comment]
This change does not require the cycle of four releases of maintaining old name?

**Comment by felixfontein** [comment]
This is kind of special, since the return value never worked as documented, we can consider this feature as "not really used". Keeping it working in 2.7.x is a courtesy to people who figured out that the result is called differently (but apparently didn't report this discrepancy).

**Comment by WojciechowskiPiotr** [comment]
shipit

**Comment by felixfontein** [comment]
@hannseman @jwitko @WojciechowskiPiotr thanks for your feedback!
@gundalow thanks for merging!

**Pull request merged**
